### DISCUSSION TITLE
Added features for Frequency Manager & Scanner modules

### DISCUSSION THREAD

**Discussion by ac6de** [AlexandreRouma/SDRPlusPlus#910] [Ideas]
Big fan of the software! A couple of cool features to build in would be:

1) Frequency Manager module - scanning between only your different saved frequencies
2) Scanner module - the ability to skip certain frequencies (or pass)
3) Scanner module - being able to define multiple bands via a low and high frequency, then scan multiple bands at once

**Comment by ac6de** [comment]
For #3 above, clarification: Being able to scan multiple bands, like the 2m band at 144 - 148 MHz and the 70cm band at 420-450 MHz, while skipping 148 - 420 MHz. Thanks again for the awesome program @AlexandreRouma .

**Comment by bnelson333** [comment]
I came here to request the same things as #1 and #2, so please add my vote!

For #1, I could see it being part of the scanner module, to say "scan my frequency range OR scan my saved frequencies", exactly like the VF/VM button on a shortwave radio.

For #2, would be handy to be able to do it while scanning. Like a blacklist/lockout button you can click and it adds to the list of bad scan frequencies that will be ignored in the next and subsequent passes.

For #3, could we have different scanlists? Like, I want my CB scan list, or my 2 meter scan list, or my 49 m SW scan list, etc. Would be nice to say, today I want to listen to my CB list, select, and hit scan, without having to reconfigure the module every time. 

I'd also add another request on top of this, #4 - the ability to force a "move on" after so many seconds. In the module today, there's a setting for how many seconds after a frequency stops broadcasting to abandon it and move on. But it won't (as far as I can tell) abandon the frequency as long as there is still something happening on it. It would be nice to have a separate setting for "seconds to abandon frequency even if there is a broadcast on it". This is how my tecsun radio works and I kinda enjoy it. That way if it's something I don't care to listen to, I know after a few seconds it'll go onto the next one, I don't always have to hit the next button. If it lands on something I do want to listen to, then I can manually stop the scan.

**Comment by cam-hyde** [comment]
Would be good if the Frequency Manager stacked the labels for bookmarks with the same frequency... its quite common to have multiple SW bookmarks with the same frequency but the labels overlay each other in the RFT display which makes it a little difficult to know what you're listening to when you hover over the label
